bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/sensor.xml

```diff
@@ -0,0 +1,240 @@
+            </channel-group>
+            <channel-group id="battery" typeId="batteryStatus">
+                <label>Battery Status</label>
+            </channel-group>
+        </channel-groups>
+
+        <config-description-ref uri="thing-type:shelly:battery" />
+    </thing-type>
+ 
+     <thing-type id="shellyflood">
+        <label>Shelly Flood (SHWT-1)</label>
+        <description>Shelly Flood sensor device (battery powered)</description>
+
+        <channel-groups>
+            <channel-group id="sensors" typeId="floodSensor">
+                <label>Sensor Data</label>
+            </channel-group>
+            <channel-group id="battery" typeId="batteryStatus">
+                <label>Battery Status</label>
+            </channel-group>
+        </channel-groups>
+
+        <config-description-ref uri="thing-type:shelly:battery" />
+    </thing-type>
+ 
+     <thing-type id="shellysense">
+        <label>Shelly Sense</label>
+        <description>Shelly Sense Remote IR controller</description>
+
+        <channel-groups>
+            <channel-group id="control" typeId="senseControl">
+                <label>Control</label>
+            </channel-group>
+            <channel-group id="sensors" typeId="senseSensors">
+                <label>Sensors</label>
+            </channel-group>
+            <channel-group id="battery" typeId="shortBatteryStatus">
+                <label>Battery Status</label>
+            </channel-group>
+        </channel-groups>
+
+        <config-description-ref uri="thing-type:shelly:battery" />
+    </thing-type>
+ 
+ 
+    <channel-group-type id="htSensor">
+       <label>Sensor Data</label>
+       <description>Data from the HT Sensor</description>
+        <channels>
+            <channel id="temperature" typeId="sensorTemp" />
+            <channel id="tempUnit"    typeId="sensorTempUnit" />
+            <channel id="humidity"    typeId="sensorHumidity" />
+            <channel id="last_update" typeId="lastUpdate" />
+            <channel id="event"       typeId="eventTrigger" />
+        </channels>
+    </channel-group-type>
+    
+    <channel-group-type id="floodSensor">
+       <label>Sensor Data</label>
+       <description>Data from the Flood Sensor</description>
+        <channels>
+            <channel id="temperature" typeId="sensorTemp" />
+            <channel id="tempUnit"    typeId="sensorTempUnit" />
+            <channel id="flood"       typeId="sensorFlood" />
+            <channel id="rainMode"    typeId="sensorRainMode" />
+            <channel id="last_update" typeId="lastUpdate" />
+            <channel id="event"       typeId="eventTrigger" />
+        </channels>
+   </channel-group-type>
+
+   <channel-group-type id="senseSensors">
+       <label>Sensors</label>
+       <description>Data from the Sense sensors</description>
+        <channels>
+            <channel id="temperature" typeId="sensorTemp" />
+            <channel id="tempUnit"    typeId="sensorTempUnit" />
```

Comment:
Please remove channel and readd as device config

Reply:
Why should we require the user to set this config in the Shelly App AND the binding config.
tempUnit is read from the device status and can be used by ruiles etc. to find out of the temperature value is in C or F. So if the user changes the device setting in the Shelly App or Web Ui the binding automatically reports this value without another config change.

Reply:
same as other comments on this channel

Reply:
see above